XML design:

sanjudavariya opened the pull request:
1. Choose Language
2. WS Loading
3. Some other necessary changes

maulik-dodia committed:
XML design:
1. Choose Language
2. WS Loading
3. Some other necessary changes

purnaPrasanth commented on app/build.gradle:
```diff
@@ -33,10 +34,15 @@ android {
     implementation 'com.github.aabhasr1:OtpView:v1.1.2-ktx' // kotlin
+
+    // Glide
+    implementation 'com.github.bumptech.glide:glide:4.11.0'
```
Add this in `baseandroid` with transitive dependency(`api`).

purnaPrasanth commented on app/build.gradle:
```diff
@@ -33,10 +34,15 @@ android {
+
+    // Glide
+    implementation 'com.github.bumptech.glide:glide:4.11.0'
+    annotationProcessor 'com.github.bumptech.glide:compiler:4.11.0'
```
replace `annotationProcessor` with `kapt`

purnaPrasanth commented on app/src/main/AndroidManifest.xml:
```diff
@@ -11,8 +11,17 @@
+            android:name=".activity.ChooseLanguageActivity"
+            android:screenOrientation="portrait">
+
+        </activity>
```
Combine the end tag with paranthesis. `<activity..... />`

purnaPrasanth commented on app/src/main/AndroidManifest.xml:
```diff
@@ -11,8 +11,17 @@
+
+        </activity>
+
+        <activity
```
https://github.com/gufranpathan/jutaav_app_dev/pull/3/files#r544535105

purnaPrasanth commented on app/src/main/java/com/jutaav/activity/ChooseLanguageActivity.kt:
```diff
@@ -15,7 +15,7 @@ class ChooseLanguageActivity :
         Timber.tag(tag()).i("${this::class.java.simpleName} Opened")
         try {
-
+            supportActionBar?.hide()
```
Handle it in style.... no need to handle in each activity.

purnaPrasanth commented on app/src/main/java/com/jutaav/activity/WSLoadingActivity.kt:
```diff
@@ -21,6 +22,10 @@ class WSLoadingActivity : BaseActivity<ActivityWSLoadingBinding>(R.layout.activi
         try {
             supportActionBar?.hide()
 
+            Glide.with(this)
```
Use `GlideModule`, gives us the flexibility to customise request and better API.

maulik-dodia committed:
XML design:
Some new XMLs, Some PR Changes, Added Styles instead of STATIC style on Widgets.

maulik-dodia committed:
1. GlideModule issue solved
2. Removed unwanted dependencies
3. Other small changes

purnaPrasanth commented:
Duplicate. Original [Here](https://github.com/gufranpathan/jutaav_app_dev/pull/4)
